src/nspawn/nspawn.c

```diff
@@ -4731,12 +4754,12 @@ static int run_container(
                 rtnl_socket_pair[2] = { -1, -1 },
                 pid_socket_pair[2] = { -1, -1 },
                 uuid_socket_pair[2] = { -1, -1 },
-                notify_socket_pair[2] = { -1, -1 },
+                fd_socket_pair[2] = { -1, -1 },
```

Comment:
EBADF, to stay consistent with your patch ;-)

Reply:
Yep.